Yosemite/Yosemite/Actions/CouponAction.swift

```diff
@@ -17,4 +17,14 @@ public enum CouponAction: Action {
                             pageNumber: Int,
                             pageSize: Int,
                             onCompletion: (Result<Bool, Error>) -> Void)
+
+    /// Deletes a coupon for a site given its ID
+    ///
+    /// - `siteID`: ID of the site that the coupon belong to.
```

Comment:
super nit: 
```suggestion
    /// - `siteID`: ID of the site that the coupon belongs to.
```

Reply:
Fixed in 0b47b3c98c2c008d59d75eb86a717651f0a9756a - thanks!